### DISCUSSION TITLE
[Esercitazione 20 dicembre 2023] Esercizio 1

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#379] [Esercizi - Viaggi/Piazza]
<p align="center">
<img  width=700 src=https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/f9b45bca-f53a-4755-be72-be24d91bcecc >
</p>

**Comment by Elia-Belli** [comment]
<p align="center">
<img width=700 src=https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/8be2c098-e7a4-4b80-ab73-4b4cab566403)>
<img width=700 src=https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/3e4d6fcb-ee4a-4f77-8a04-e18e30a03880>
<img width=700 src=https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/3ea2ac6d-cb13-4697-b586-44d2139861ef>
<img width=700 src=https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/06e4a94b-14a8-48cf-bdb3-64869679f567>
<img width=700 src=https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/5cb2eb62-a585-4686-91b4-68f7df3904f3>
<img width=700 src=https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/3ccd0bc2-e0aa-48ae-a4af-87b1a2fed5f1>
<img width=700 src=https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/1bbca8da-6f71-4d94-8d9e-2e8a2fb096c5>

<img width=700 src=https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/08099b16-21ed-4b45-a279-54ad882342e4>
</p>

**Comment by cornflexxx** [comment]
![sol1_6](https://github.com/sapienzastudentsnetwork/algebra2324/assets/121803863/6ae76ca2-43bd-49bc-a83b-73efb099a402)

**Comment by CasuFrost** [comment]
![Esercitazione 4-1](https://github.com/sapienzastudentsnetwork/algebra2324/assets/97389762/2ee753ee-f4a1-4a51-a6b7-e91c7eb034be)
